Samples/kmp-app/shared/build.gradle.kts

```diff
@@ -17,23 +17,28 @@ kotlin {
         homepage = "Link to the Shared Module homepage"
         ios.deploymentTarget = "14.1"
         podfile = project.file("../iosApp/Podfile")
+
+        pod("Sentry", "~> 7.19.0")
+
         framework {
             baseName = "shared"
+            export("io.sentry.kotlin.multiplatform:sentry-kotlin-multiplatform:0.0.1")
         }
     }
-    
+
     sourceSets {
-        val commonMain by getting
+        val commonMain by getting {
+            dependencies {
+                api("io.sentry.kotlin.multiplatform:sentry-kotlin-multiplatform:0.0.1")
```

Comment:
Still has to depend on the project, we can solve in. different PR

Reply:
yes I will work on that in the next one